src/InheritDoc/package/build/SauceControl.InheritDoc.targets

```diff
@@ -3,24 +3,27 @@
 
 	<PropertyGroup>
 		<InheritDocEnabled Condition="'$(InheritDocEnabled)'=='' and '$(Configuration)'!='debug' and '$(DesignTimeBuild)'!='true' and '$(BuildingForLiveUnitTesting)'!='true'">true</InheritDocEnabled>
+
+		<!-- Let the InheritDocTask work as a replacement for the task copying the documentation file to the output. -->
+		<CopyDocumentationFileToOutputDirectory Condition="'$(InheritDocEnabled)'=='true'">false</CopyDocumentationFileToOutputDirectory>
 	</PropertyGroup>
 
-	<ItemGroup>
-		<_InheritDocBackupFile Include="@(DocFileItem -> '%(RootDir)%(Directory)%(Filename).original%(Extension)')" />
+	<ItemGroup Condition="'$(InheritDocEnabled)'=='true' and '@(DocFileItem)'=='@(FinalDocFile)'">
+		<!-- Force intermediate documentation file in order to enable incremental build. -->
+		<DocFileItem Remove="@(DocFileItem)"/>
+		<DocFileItem Include="@(FinalDocFile->'$(IntermediateOutputPath)%(Filename)%(Extension)')"/>
 	</ItemGroup>
 
-	<Target Name="_InheritDocPostProcess" AfterTargets="CoreCompile" DependsOnTargets="ResolveAssemblyReferences;CoreCompile;_CheckForCompileOutputs" Inputs="@(DocFileItem)" Outputs="@(_InheritDocBackupFile)" Condition="'$(InheritDocEnabled)'=='true' and '$(_DocumentationFileProduced)'=='true'">
+	<Target Name="_InheritDocPostProcess" AfterTargets="CoreCompile" DependsOnTargets="ResolveAssemblyReferences;CoreCompile;_CheckForCompileOutputs" Inputs="@(DocFileItem)" Outputs="@(FinalDocFile)" Condition="'$(InheritDocEnabled)'=='true' and '$(_DocumentationFileProduced)'=='true'">
 
 		<Message Importance="normal" Text="InheritDoc processing file: @(DocFileItem -> '%(Filename)%(Extension)')" />
 
-		<Copy SourceFiles="@(DocFileItem)" DestinationFiles="@(_InheritDocBackupFile)" />
-
 		<ItemGroup>
-			<FileWrites Condition="Exists('@(_InheritDocBackupFile)')" Include="@(_InheritDocBackupFile)" />
+			<FileWrites Condition="Exists('@(FinalDocFile)')" Include="@(FinalDocFile)" />
```

Comment:
This will no longer work as intended, as it previously ran after the `Copy` task, which would have created the `_InheritDocBackupFile` it was conditionally adding to `FileWrites`.

Since the `FinalDocFile` is now created by `InheritdocTask`, the update to `FileWrites` needs to happen after that.